packages/core/src/createActor.ts

```diff
@@ -178,6 +181,12 @@ export class Actor<TLogic extends AnyActorLogic>
           );
         }
         (child as any)._stop();
+      },
+      emit: (emittedEvent) => {
+        for (const handler of this.eventListeners.get(emittedEvent.type) ??
```

Comment:
This should use `.slice()` to make the copy of the **current** listeners. We don't want to notify the listeners that might get added while notifying this listeners list etc

Reply:
Perhaps not `.slice` since that's a Set from what I can see but you know what I mean :p